### PR TITLE
D8CORE-2653: drop shadow on load more button

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Adding a drop shadow to the load more buttons for people, events, news on the SOE

# Review By (Date)
- 9/25

# Criticality
- Low
- Events, people, news on SOE

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Generate lots of content for New, Events and People.
3. View the buttons for Load More on News, Events and People. See the drop shadow. 
4. View the see all button on a news/event node page. See the drop shadow.

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- SoE theme for the News, People, Events

# Associated Issues and/or People
- D8CORE-2653

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/scss/theme/_button.scss [hunk 1]
```diff
@@ -36,5 +36,12 @@
   }
 }
 
+// Back to all events or news link.
+.more-link {
+  a {
+    @include box-shadow('shallow');
+  }
+}
```
Comment (sherakama) [inline src/scss/theme/_button.scss:44]: Is this really a button and do you want the drop shadow everywhere there is a `more-link` class?

Comment (jenbreese) [reply src/scss/theme/_button.scss:44]: It is really a button but not in HTML. Maybe I should change the HTML? `<div class="more-link"><a href="/news">See All News</a></div>`

**Review by sherakama — APPROVED** [review]
ok

**Review by imonroe — APPROVED** [review]
OK by me.

**Pull request merged**